### DISCUSSION TITLE
Vorta - Advanced - Extra Borg Arguments

### DISCUSSION THREAD

**Discussion by MacH59-cos** [borgbase/vorta#1213] [General]
Vorta - Advanced - Extra Borg Arguments

Can this be used for --storage-quota 2T, --restrict-to-path or --restrict-to-repository arguments??

Several arguments separated by comma or space??

Can the argument be changed later for instance --storage-quota 5T ??

Can an argument be added later??

Please explain because there is no specific information about this anywhere, as far as I can tell...

**Comment by m3nu** [comment]
Separated by spaces.

We could update the placeholder example to make it clearer.

**Reply by MacH59-cos** [reply]
On Vorta for Mac there is no placeholder example (or should there be?)

Could you or someone else please answer my other questions?? I just looked at it again and now believe that you can use this (Extra Borg Arguments) only once namely when you create a new repository. That then means you can not change the setting for the quota afterwards and you can not instigate a quota when the repository was created without a quota? Correct?

**Comment by real-yfprojects** [comment]
> Can this be used for --storage-quota 2T, --restrict-to-path or --restrict-to-repository arguments??

You can only supply arguments to the `create` command.

> Several arguments separated by comma or space??

Space as m3nu already said.

> Can the argument be changed later for instance --storage-quota 5T ??
> Can an argument be added later??

Yes. But only for `create` commands that are run afterwards.

> Please explain because there is no specific information about this anywhere, as far as I can tell...

The label of the entry in question gives a hint through the use of the phrase `when creating a backup` as well as through the link to the `borg create` docs. But I do agree that it could be clearer.
